package.json

```
@@ -227,11 +227,11 @@
     "lint:js": "eslint static/app tests/js --ext .js,.jsx,.ts,.tsx",
     "lint:css": "stylelint 'static/app/**/*.[jt]sx'",
     "lint:biome": "biome check .",
-    "lint:prettier": "prettier \"**/*.md\" \"**/*.yaml\" \"**/*.{ts,tsx}\" --check",
+    "lint:prettier": "prettier \"**/*.md\" \"**/*.yaml\" \"**/*.[jt]s?(x)\" --check",
```

i think you want this with the x being optional. might just be easier to do `{js,ts,jsx,tsx}`
```suggestion
    "lint:prettier": "prettier \"**/*.md\" \"**/*.yaml\" \"**/*.[jt]s(x)?\" --check",
```

---

i had `{js,ts,jsx,tsx}` but it was complaining in one of the repos that no files were found. So i wanted to do the confusing regexp thing to keep the repos the same, and get full coverage if a new file (like a script) pops up